packages/aws-client/src/Lambda.ts

```diff
@@ -2,12 +2,26 @@
  * @module Lambda
  */
 import { InvocationType } from '@aws-sdk/client-lambda';
+import { EventBridgeEvent } from 'aws-lambda';
 import Logger from '@cumulus/logger';
 import { lambda } from './services';
 import { inTestMode } from './test-utils';
 
 const log = new Logger({ sender: 'aws-client/Lambda' });
 
+export type StepFunctionEventBridgeEvent = EventBridgeEvent<'Step Functions Execution Status Change', { [key: string]: string }>;
+
+/**
+ * Bare check for EventBridge shape
+ *
+ * @param {{ [key: string]: any }} event
+ * @returns {event is EventBridgeEvent}
+ */
+export const isEventBridgeEvent = (event: Object): event is StepFunctionEventBridgeEvent => (
+  event instanceof Object
+  && 'detail' in event
+);
```

Comment:
Why is `event instanceof Object` needed here?    Is the intent to guard against someone ignoring typechecking in the raw .js? 

I... think I agree with the defensive intent here, but it might warrant a comment (or unit if that's not already demanded by coverage metrics).

Reply:
precisely: not knowing where this might be used in a js file.